### PR TITLE
Simplify local install instructions

### PR DESCRIPTION
**Pull request by karlkfi**
## Description
Simplify local install instructions

## Urgency
- [ ] Blocker 
- [ ] High
- [X] Medium

### CODE REVIEW

File: 1.7/administration/installing/local.md [hunk 1]
```diff
@@ -4,24 +4,12 @@ nav_title: Local
 menu_order: 3
 ---
 
-This installation method uses Vagrant to create a cluster of virtual machines on your local machine that can be used for demos, development, and testing with DC/OS.
+DC/OS Vagrant uses Vagrant and VitualBox to create a cluster of virtual machines on your local machine that can be used for demos, development, and testing with DC/OS.
```
Comment (joel-hamill) [inline 1.7/administration/installing/local.md]: `VirtualBox`

File: 1.7/administration/installing/local.md [hunk 2]
```diff
@@ -4,24 +4,12 @@ nav_title: Local
...
+Quick Start and full documentation are maintained in the dcos-vagrant GitHub repository.
 
 - For the latest bug fixes, use the [master branch](https://github.com/dcos/dcos-vagrant/).
 - For increased stability, use the [latest official release](https://github.com/dcos/dcos-vagrant/releases/latest/).
 - For older releases on DC/OS, you may need to download an [older release of DC/OS Vagrant](https://github.com/dcos/dcos-vagrant/releases/).
 
-Follow the the deploy instructions to set up your host machine correctly and to install DC/OS.
-
-Note that you will require at least 5GB of free memory to run DC/OS via this method.
+Feel free to try the master branch first. It's relatively stable and easiest to update (git pull). Then try the official releases if the master branch doesn't work for you.
```
Comment (joel-hamill) [inline 1.7/administration/installing/local.md]: Use command syntax for `git pull`?

File: 1.8/administration/installing/local.md [hunk 3]
```diff
@@ -4,24 +4,12 @@ nav_title: Local
 menu_order: 3
 ---
 
-This installation method uses Vagrant to create a cluster of virtual machines on your local machine that can be used for demos, development, and testing with DC/OS.
+DC/OS Vagrant uses Vagrant and VitualBox to create a cluster of virtual machines on your local machine that can be used for demos, development, and testing with DC/OS.
```
Comment (joel-hamill) [inline 1.8/administration/installing/local.md]: See comments above.

File: 1.7/administration/installing/local.md (hunk 1 above)
Comment (karlkfi) [inline 1.7/administration/installing/local.md]: fixing

File: 1.8/administration/installing/local.md (hunk 3 above)
Comment (karlkfi) [inline 1.8/administration/installing/local.md]: fixing

File: 1.7/administration/installing/local.md (hunk 2 above)
Comment (karlkfi) [inline 1.7/administration/installing/local.md]: sure.

**Comment by karlkfi** [comment]
Feedback addressed.

**Pull request merged**